From 643096b69c2d2c3d753fe89a11ad1d20a77b6901 Mon Sep 17 00:00:00 2001
Message-Id: <RFC-patch-2.5-643096b69c2-20230317T042408Z-avarab@gmail.com>
In-Reply-To: <RFC-cover-0.5-00000000000-20230317T042408Z-avarab@gmail.com>
References: <pull.1463.v2.git.git.1678925506.gitgitgadget@gmail.com>
	<RFC-cover-0.5-00000000000-20230317T042408Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 17 Mar 2023 03:15:26 +0100
Subject: [RFC PATCH 2/5] config.c: use "enum config_origin_type", not "int"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the current_config_origin_type() function to use the
appropriate enum type for its "type" variable.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 config.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config.c b/config.c
index 00090a32fc3..a4105c456c3 100644
--- a/config.c
+++ b/config.c
@@ -3768,7 +3768,8 @@ int parse_config_key(const char *var,
 
 const char *current_config_origin_type(void)
 {
-	int type;
+	enum config_origin_type type;
+
 	if (current_config_kvi)
 		type = current_config_kvi->origin_type;
 	else if(cf)
-- 
2.40.0.rc1.1034.g5867a1b10c5

